app/controllers/volunteer_ops_controller.rb

```diff
@@ -75,6 +87,27 @@ def build_map_markers(organisations)
     end
   end
 
+  # This needs to be a helper method for the do-it API
+  def collect_all_items (host, href, orgs)
+    n=1
+    while href do
+      url = host + href
+      response = HTTParty.get(url)
+      if response.body && response.body != '[]'
+        respItems = JSON.parse(response.body)["data"]["items"]
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)<br>Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)